_data/operations.yml

```diff
@@ -46,7 +46,6 @@ members:
   country: USA
   github: wdduncan
   groups:
-  - editorial
```

Comment:
What does it mean to have someone on the Ops committee with no assigned groups? Don't all active Ops members have to belong to one or more groups?

Reply:
That's my understanding but my memory is fuzzy on that. Heck, there are many members listed that never join meetings. Anyway, this is to address the EWG issue only. Anything beyond that is out of my scope and will need to be brought up at an Ops meeting.

Reply:
I'll add it to the agenda

Reply:
This is why the PR is failing. If a person doesn't have a group assigned to him or her, you have to remove the `groups:` key, too.

Reply:
Thank you @anitacaron !

Reply:
@anitacaron Either I did something wrong, or there is something else causing the failure. I fixed the issue you mentioned but things are still failing. Perhaps I should just start over.